cylc/flow/suite_db_mgr.py

```diff
@@ -294,6 +294,11 @@ def put_suite_params(self, schd):
             final_point_str = str(schd.config.final_point)
         self.db_inserts_map[self.TABLE_SUITE_PARAMS].extend([
             {"key": self.KEY_UUID_STR, "value": str(schd.uuid_str)},
+            {"key": self.KEY_INITIAL_CYCLE_POINT, "value":
+                str(schd.config.initial_point)},
+            {"key": self.KEY_START_CYCLE_POINT, "value":
+                str(schd.config.start_point)},
+            {"key": self.KEY_FINAL_CYCLE_POINT, "value": final_point_str},
```

Comment:
initial point and final point here were removed by @matthewrmshin in #3147 - I haven't gone back to read the discussion there (I'm in a workshop) - but he was probably right!

Reply:
Got it, in that case we can probably remove the part qhere the keys are still used here.

Reply:
Ah! Mea culpa! I believe in the families PR I changed the code to build the workflow tree for the first workflow (implemented that for a single workflow, when looking at the `/vuews/Tree.vue`).

Easy to cover with an e2e test. It might be time to start considering adding more e2e tests, and adding them to Travis/CI.

Reply:
I think these were intentionally omitted, will have to do a dive through the logic to find out why.

What was the motive for this change? Were you experiencing issues with restart?

Reply:
Nope, I can remove the initial and the final point.

The `final_stop_point` variable however, is never used.